test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -28,37 +27,62 @@ const defaultProps = {
 
 describe('<AddressSection>', () => {
   it('should render', () => {
-    const tree = SkinDeep.shallowRender(<AddressSection {...defaultProps}/>);
+    const tree = SkinDeep.shallowRender(<AddressSection { ...defaultProps }/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.exist;
   });
 
+  it('should display an address if one is provided in props', () => {
+    const tree = SkinDeep.shallowRender(<AddressSection { ...defaultProps }/>);
+    const addressContent = tree.dive(['AddressContent']);
+    expect(addressContent).to.exist;
```

Comment:
Sorry to be such a pain... 😬

Reply:
My thinking with these is that we want to make sure each child component is getting rendered, and should be testing _what_ gets rendered in the actual unit tests for that subcomponent. Seem like you don't agree 😛 (which is completely fine), but just wanted to get clarification on your thinking.